remoteappmanager/static/js/gamodule.js

```diff
@@ -7,46 +7,31 @@ define([
     "use strict";
 
     function init() {
-        var module;
-
         if (window.apidata.analytics !== undefined) {
             window.ga('create', window.apidata.analytics.tracking_id, 'auto');
         } else {
-            window.ga = function () {
-            };
+            window.ga = function() {};
         }
-        module = function () {
+
+        return function() {
             window.ga.apply(this, arguments);
         };
-        return module;
     }
 
-    var GaView = Vue.extend({
-        beforeCreate: function() {
-            this.ga = init();
-        },
-
-        computed: {
-            current_app: function() {
-                return this.model.app_list[this.model.selected_index] || null;
-            }
-        },
+    var GaObserver = function() {
+        this.ga = init();
+    };
 
-        watch: {
-            'current_app.status': function() {
-                if(this.current_app.is_starting()) {
-                    this.ga("send", "event", {
-                        eventCategory: "Application",
-                        eventAction: "start",
-                        eventLabel: this.current_app.app_data.image.name
-                    });
-                }
-            }
-        }
-    });
+    GaObserver.prototype.trigger_application_starting = function(name) {
+        this.ga("send", "event", {
+            eventCategory: "Application",
+            eventAction: "start",
+            eventLabel: name
+        });
+    };
 
     return {
         init: init, // For testing purpose
```

Comment:
at this point, init could be a method of GaObserver. Keep it separate, because we still need to patch it at the test level